lib/libc/termios/lib_loopback_en.c

```diff
@@ -0,0 +1,68 @@
+/****************************************************************************
+ *
+ * libc/termios/lib_loopback_en.c
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <sys/ioctl.h>
+#include <termios.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Variables
+ ****************************************************************************/
+
+/****************************************************************************
+ * Public Variables
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: loopback_en
+ *
+ * Description:
+ *   calls the ioctl to enable/ disable the loopback mode of UART
+ *
+ * Input Parameters:
+ *   fd  - The 'fd' argument is an open file descriptor associated with a terminal.
+ *   cmd - ioctl argument.
+ *   termiosp - The termiosp argument is a pointer to a termios structure.
+ *
+ * Returned Value:
```

Comment:
Is there any reason that return type of this function is `void`?
A `ioctl` executes some parameters like fd, args for control something.
The result depends on incoming parameters so it can be failed according to them.
At that time, we can't know whether enabling or disabling loopback is succeed or not.
Please change return type from `void` to `int` to be like ioctl if there is no special reason.

Reply:
corrected